README.md

```diff
@@ -233,6 +235,21 @@ Need to setup Flatpak (https://flatpak.org/setup/):
 * Chrome OS
 
 
+Systemd
+-------
+If your distribuition is running under systemd and you want to generate and upload
+hw-probe report autmaticaly, please install as root:
+
+    cp -a systemd/* $(systemdsystemunitdir)/
```

Comment:
bash: systemdsystemunitdir: command not found

Reply:
Yes i know there is not such variable $(systemdsystemunitdir) as systemd does not install anything like on env.
I may update this and use `/path/to/systemd/system` or use `$(pkg-config --variable=systemdsystemunitdir systemd)` ofcourse pkg-config should be installed and systemd.pc file must be present in pkg-config directory :)